pilot/cmd/cmd.go

```diff
@@ -50,6 +50,22 @@ var VersionCmd = &cobra.Command{
 	},
 }
 
+// AddFlags carries over glog flags with new defaults
+func AddFlags(rootCmd *cobra.Command) {
```

Comment:
@kyessenov this seems to be necessary for the following tests (possibly others as well):

- pilot-integration-auth
- pilot-integration-noauth
- e2e-smoke.sh

Any idea why this is necessary? and how we might be able to make the tests pass without it?

Reply:
@kyessenov after our offline discussion, I tried reducing this logic to just `rootCmd.PersistentFlags().AddGoFlag(gf)`, but tests are again failing ... it is apparently not sufficient. So somehow the glog-specific logic is needed for our tests.  Any ideas? Or any idea who might know what's going on?